README.md

```diff
@@ -60,20 +60,34 @@ The process of decision making in this collection is based on discussing and fin
 
 Every voice is important. If you have something on your mind, create an issue or dedicated discussion and let's discuss it!
 
-## Tested with Ansible
+## Tested with Ansible and the following Python versions
 
+Tested Ansible versions:
 - 2.9
 - 2.10
 - 2.11
 - 2.12
 - devel
 
+Tested Python versions:
+- 2.6
+- 2.7
+- 3.5
+- 3.6
+- 3.7
+- 3.8
+- 3.9
+
+In fact of SAP licensing and hardware requirements integration tests are hard to execute. 
```

Comment:
vielleicht besser: Due to SAP licensing and hardware requirements, integration tests are momentarily not feasible.

Reply:
Vielleicht sollten wir noch ein Satz dazu schreiben, dass die Module auf SAP-Systemen getestet wurden?